### DISCUSSION TITLE
Request organization name of legacy account due to lost login

### DISCUSSION THREAD

**Discussion by srimanachanta** [community/community#143156] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I am the CTO of a nonprofit organization and we currently use GitHub to host all of our codebase using a free organization. In years prior, a previous member created an organization under the name of our nonprofit, but then left, leaving the name taken on GitHub. Subsequently, we have been forced to take a similar yet different name for our current organization name, which we feel doesn't reflect our nonprofit's name properly. Would it be possible to have this other organization's name changed, deleted, or have control of it relinquished to us? I can provide relevant documents proving our ownership over all associated domains and name materials.

**Comment by ebndev** [comment, marked as answer]
Hi @srimanachanta,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
